text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -32,15 +32,16 @@ set(TARGET_NAME openvino_continuous_batching)
 add_library(${TARGET_NAME} STATIC
     src/tokenizer.cpp
     src/generation_config.cpp
+    src/generation_handle.cpp
     src/continuous_batching_pipeline.cpp
     src/paged_attention_transformations.cpp)
 
 add_library(openvino::continuous_batching ALIAS openvino_continuous_batching)
 
 target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src"
                                           PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
-target_compile_definitions(${TARGET_NAME} PRIVATE OPENVINO_TOKENIZERS_PATH=\"$<TARGET_FILE:openvino_tokenizers>\")
-set_target_properties(${TARGET_NAME} PROPERTIES CXX_STANDARD 11 CXX_STANDARD_REQUIRED ON)
+target_compile_definitions(${TARGET_NAME} PRIVATE OPENVINO_TOKENIZERS_PATH="libopenvino_tokenizers.so")
```

Comment:
maybe we can use `TARGET_FILE_NAME` generator expression instead? 
I suppose it fill provide platform intendent file name